llvm/lib/Support/Unix/Signals.inc

```diff
@@ -80,7 +80,8 @@
 
 using namespace llvm;
 
-static void SignalHandler(int Sig);     // defined below.
+static void SignalHandler(int Sig, siginfo_t *Info,
```

Comment:
Drop unnecessary "defined below".

(If the declaration is unused, clang -Wall will report a warning.)